### PR TITLE
Kathleen Cannell bio added to README

### PR DESCRIPTION
**Pull request by kathleencannell**
I added a brief bio about myself to the README. I used some italics, some bold lettering, and I added an image to represent my major. I also embedded links to some relevant pages. I used the Markdown syntax to do these things, and it was not too difficult. This first bit of coding was not as scary as I thought it would be.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,19 @@
-# hw01
\ No newline at end of file
+# hw01
+
+My name is *Kathleen Cannell*. I just finished my second year in the college.
+I am majoring in Economics:
+![Economics](https://i.kym-cdn.com/entries/icons/facebook/000/029/959/Screen_Shot_2019-06-05_at_1.26.32_PM.jpg)
```
Comment (wanitchayap) [inline README.md:5]: Nice picture 😆

File: README.md [hunk 2]
```diff
@@ -1 +1,19 @@
...
\ No newline at end of file
+# hw01
+
+My name is *Kathleen Cannell*. I just finished my second year in the college.
+I am majoring in Economics:
+![Economics](https://i.kym-cdn.com/entries/icons/facebook/000/029/959/Screen_Shot_2019-06-05_at_1.26.32_PM.jpg) 
+
+I hope to minor in Inequality, Social Problems, and Change.
+
+**Background**
```
Comment (wanitchayap) [inline README.md:9]: You need at least 2 spaces for markdown to actually start a new line for you. Alternatively, you can use a small header here too (more #s)

**Review by wanitchayap — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |    X       |              |            |
| **Presentation**            |           |       X       |            |
| **Achievement, creativity** |           |      X        |            |
| **Reproducibility**         |    X       |              |            |

Remarks:
* Great job!
* Descriptive commit messages 👍

File: README.md (hunk 1 above)
Comment (IanGGG) [inline README.md:5]: I like the pic!

File: README.md [hunk 3]
```diff
@@ -1 +1,19 @@
...
+I am from [Columbus, Ohio](https://en.wikipedia.org/wiki/Columbus,_Ohio).
+
+**Current Information**
+I am back home in Columbus, and am in the Summer Institute for Social Research Methods.
+I am excited to start research with Professor Waldo Johnson Jr. of SSA.
+
+**Interests**
+I love music and concerts of all kinds. Last year I got to see Clairo, Logic, Khalid, and [Dominic Fike](https://www.youtube.com/watch?v=sMYcubNOrPs), along with a few smaller shows.
+I also enjoy reading, knitting, and being outside.
```
Comment (IanGGG) [inline README.md:18]: Maybe you can try to use a list here( it might not be necessary, but it will be a good try :) )

File: README.md [hunk 4]
```diff
@@ -1 +1,19 @@
-# hw01
\ No newline at end of file
+# hw01
+
```
Comment (IanGGG) [inline README.md:2]: Maybe you can add a header here by using ## XXXX

File: README.md (hunk 2 above)
Comment (kathleencannell) [reply README.md:9]: Thank you!

**Comment by sandramauro** [comment]
I like your use of links in the text. I also like that you used bolded font to break up sections in your text. Also, nice image! I think you could have used more headers to break up the sections.

You could have considered using a list. Here is what I used to make a list:

+ text
+ text
+ text

**Pull request closed without merging**